src/apis/data_api.ts

```diff
@@ -51,20 +51,17 @@ export async function GetPieCostChartData(location_id: string) {
   return await ApiWrapper(response, true);
 }
 
-export async function GetLinePlotDataByLocation(
-  location_id: string,
-  lineType: string,
-) {
-  const response = await fetch(`${backend}/lineplot_data/${location_id}`, {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
+export async function GetLinePlotDataByLocation(location_id: string) {
+  const response = await fetch(
+    `${backend}/get_logs_for_plot_by_location/${location_id}`,
+    {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      credentials: "include",
     },
-    credentials: "include",
-    body: JSON.stringify({
-      lineType: lineType,
-    }),
-  });
+  );
```

Comment:
If we create an API call we should alawys create error handling inside of it.

Reply:
Now we have error handling in the ApiWrapper that we call in each api call.